proxy_config

```diff
@@ -0,0 +1,32 @@
+# This is a comment.
+
+port 3030;  # This is also a comment.
+
+path /static StaticRequestHandler {
+  root /test;
+}
+
+# Empty config blocks are allowed; not all handlers need options
+# Currently: no default options for handlers
+path /echo EchoRequestHandler {}
+
+path /status StatusHandler {}
+
+# Proxy request to www.ucla.edu/academics/ <= that forward slash is required
```

Comment:
To access www.ucla.edu/academics directly, the site redirects it to www.ucla.edu/academics/. Since the purpose of this config was to handle a subpath (not test redirects), I added the the forward slash there for convenience